src/tests/system/tests/test_sss_override.py

```diff
@@ -0,0 +1,246 @@
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.AD)
+def test_local_overrides__user(client: Client, provider: GenericProvider):
+    """
+    :title: Local override user account
+    :setup:
+        1. Create posix user "user1" with posix attributes defined
+        2. Start SSSD
+    :steps:
+        1. Override "user1" to "o-user1"
+        2. Authenticate as "user1", short and fully qualified name
+        3. Authenticate as "o-user1", short and fully qualified name
+        4. Override posix attributes for "o-user1"
+        5. Search for user override
+        6. Delete override
+    :expectedresults:
+        1. User local override is created
+        2. Authentication successful for both short and fully qualified name
+        3. Authentication successful for both short and fully qualified name
+        4. Local override posix attribute updated
+        5. Local override "o-user1" found
+        6. Local override is deleted
+    :customerscenario: False
+    """
+    provider.user("user1").add(
+        uid=999011, gid=999011, home="/home/user1", gecos="user", shell="/bin/bash", password="Secret123"
+    )
+    client.sssd.start()
+    client.sss_override.user("user1").add(name="o-user1")
+    client.sssd.restart()
+
+    assert client.ssh("user1", "Secret123")
+    assert client.ssh("o-user1", "Secret123")
+    assert client.ssh(f"user1@{client.sssd.domain}", "Secret123")
+    assert client.ssh(f"o-user1@{client.sssd.domain}", "Secret123")
+
+    assert client.tools.getent.passwd("user1").uid != 999999
+    assert client.tools.getent.passwd("user1").gid != 888888
```

Comment:
It would be better to check that it equals to the value set in user("user1").add()

Also avoid running the command repeatedly, but run it only once:
```python
result = client.tools.getent.passwd("user-1")
assert result.uid ...
assert result.gid ...
```

Reply:
Same on other places.